.gitignore

```diff
@@ -14,3 +14,5 @@ yarn.lock
 examples/config.js
 coverage.html
 .nyc_output
+
+lib/.DS_Store
```

Comment:
This should be able to turn into a generic `.DS_Store` without the `lib/` part in front